### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by taleldayekh**

### CODE REVIEW

File: raiden-ts/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [Unreleased]
```
Comment (andrevmatos) [inline raiden-ts/CHANGELOG.md:3]: In the bottom of the changelog files there's a line with the link to the github compare between versions, you need to bump them as well

Comment (taleldayekh) [reply raiden-ts/CHANGELOG.md:3]: Ah, I see.

File: raiden-dapp/CHANGELOG.md [hunk 2]
```diff
@@ -175,7 +175,7 @@
 - Add link to privacy policy.
 - Add basic transfer screen.
 
-[Unreleased]: https://github.com/raiden-network/light-client/compare/v0.5.0...HEAD
+[0.5.1]: https://github.com/raiden-network/light-client/compare/v0.5.0...HEAD
```
Comment (andrevmatos) [inline raiden-dapp/CHANGELOG.md:178]: Head is just for Unreleased, and you can leave the unreleased (updated) link
```suggestion
[Unreleased]: https://github.com/raiden-network/light-client/compare/v0.5.1...HEAD
[0.5.1]: https://github.com/raiden-network/light-client/compare/v0.5.0...v0.5.1
```

File: raiden-ts/CHANGELOG.md [hunk 3]
```diff
@@ -136,7 +136,7 @@
 - Add protocol message implementation.
 
 
-[Unreleased]: https://github.com/raiden-network/light-client/compare/v0.5.0...HEAD
+[0.5.1]: https://github.com/raiden-network/light-client/compare/v0.5.0...HEAD
```
Comment (andrevmatos) [inline raiden-ts/CHANGELOG.md:139]: Same

File: raiden-dapp/CHANGELOG.md [hunk 4]
```diff
@@ -175,7 +175,7 @@
 - Add link to privacy policy.
 - Add basic transfer screen.
 
-[0.5.1]: https://github.com/raiden-network/light-client/compare/v0.5.0...HEAD
```
Comment (andrevmatos) [inline raiden-dapp/CHANGELOG.md]: `0.5.1` line is still missing

File: raiden-ts/CHANGELOG.md [hunk 5]
```diff
@@ -136,7 +136,7 @@
 - Add protocol message implementation.
 
 
-[0.5.1]: https://github.com/raiden-network/light-client/compare/v0.5.0...HEAD
```
Comment (andrevmatos) [inline raiden-ts/CHANGELOG.md]: Same

**Pull request merged**